test/maintainance/Maintenance.test.ts

```diff
@@ -114,16 +124,16 @@ describe('Maintenance test', () => {
     await network.provider.send('hardhat_setCoinbase', [coinbase.address]);
 
     localEpochController = new EpochController(minOffsetToStartSchedule, numberOfBlocksInEpoch);
-    await localEpochController.mineToBeforeEndOfEpoch();
+    await localEpochController.mineToBeforeEndOfEpoch(2);
```

Comment:
Idk why it needed to be set to `2`. Before this PR, this number is `1` and the code ran well.